### PR TITLE
Adding everywhere parameter to navigator.id.logout

### PR DESCRIPTION
**Pull request by ozten**
`navigator.id.logout()` remains the same.

`navigator.id.logout({everywhere: true})` clears the server side session as well as the `localStorage.loggedIn` status.

### CODE REVIEW

File: resources/static/communication_iframe/start.js [hunk 1]
```diff
@@ -92,6 +92,8 @@
   });
 
   chan.bind("logout", function(trans, params) {
+    if (params && params.everywhere) user.logoutUser();
```
Comment (shane-tomlinson) [inline resources/static/communication_iframe/start.js:95]: @ozten - is this something that should be exposed to _all_ RPs or only the marketplace? Do we want to allow any RP to specify that you should be logged out on all sites? That seems like it leaves a lot of room open for user confusion.

Comment (ozten) [inline resources/static/communication_iframe/start.js:95]: My understanding is that it is for all RPs based on @benadida's requirements.

Comment (callahad) [inline resources/static/communication_iframe/start.js:95]: As the designated end-user curmudgeon, this fells bad / strange / confusing to me. Has Crystal reviewed this?

Edit: Wait, unless I'm misunderstanding what this does...

Edit: Also, this is against the **b2g branch**. So all of my knowledge / objections / etc go out the window :)

**Comment by ozten** [comment]
Here is the [Marketplace bug#808831](https://bugzilla.mozilla.org/show_bug.cgi?id=808831)

**Comment by shane-tomlinson** [comment]
@ozten - this needs to be updated before merge. And like @callahad - this is against the B2G branch so all of my objections are nullified.

**Comment by ozten** [comment]
I think [Bug#808831 Comment#3](https://bugzilla.mozilla.org/show_bug.cgi?id=808831#c3) makes this PR obsolete.

**Pull request closed without merging**